settings.py

```diff
@@ -27,7 +27,7 @@ def get_key(key_name):
 PRIVATE_KEY_PASSWORD = os.getenv("PRIVATE_KEY_PASSWORD", "digitaleq")
 
 
-POSIE_URL = os.getenv('POSIE_URL', 'http://posie:5000')
```

Comment:
As discussed offline, can we change this to use the docker host instead of a hardcoded ip like http://sdx-validate:5000/validate'?